buildengine/engine.go

```diff
@@ -25,6 +25,20 @@ import (
 	"github.com/TBD54566975/ftl/internal/rpc"
 )
 
+type CompilerBuildError struct {
+	err error
+}
+
+func (e CompilerBuildError) Error() string {
+	return e.err.Error()
+}
+
+func (e CompilerBuildError) Is(target error) bool {
```

Comment:
I think it would be simpler just to implement `Unwrap() error { return e.err }` - the `errors` package functions will do the correct thing.